xdp-dump/xdpdump.h

```diff
@@ -9,7 +9,7 @@
 /******************************************************************************
  * General definitions
  ******************************************************************************/
-#define PERF_MMAP_PAGE_COUNT	16
+#define PERF_MMAP_PAGE_COUNT	256
```

Comment:
Why is this changing?

Reply:
With 16 it was easy to run out of perf buffer space, so I increased it to a decent number.

Reply:
Right, OK. Just curious :)